images/base/scripts/common/setup.sh

```diff
@@ -93,7 +85,7 @@ fi
 cd /tmp
 git clone https://github.com/google/protobuf.git
```

Comment:
I had thought of that, but it would only support x86 so I was thinking src-release was better in this case.  I was thinking since this image gets cached it had a negligible performance impact.  I could be convinced otherwise.